### ISSUE TITLE
Missing Accessibility Report button with @ladle/react v3.1.0 and up

### ISSUE THREAD

**Issue by esbeto** [tajo/ladle#527]
**Describe the bug**

For some reason I lost the accessibility report and button when upgrading ladle, I tried different versions and the last one that works for me is 3.0.0

**Reproduction**

I was able to reproduce with [Stackblitz](https://ladle.dev/new). 
Just change the dependency to:
```
    "@ladle/react": "3.0.0",
```
to see the accessibility report button.

**Environment**

- OS: MacOS Ventura 13.6
- Browser: Firefox 118 Chrome 118
- (I wasn't aware both browsers are now in matching version numbers 😅)

**Comment by tajo** [comment]
It's just disabled by default now. You can easily enable it again: https://ladle.dev/docs/a11y

`axe-core` is a pretty big dependency so we don't want to bundle it with Ladle unless it's actively used.

**Comment by esbeto** [comment]
Thanks! I suggest to include changes in defaults in the release notes next time.
